src/cascadia/TerminalConnection/ConnectionInformation.cpp

```diff
@@ -36,21 +43,58 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
         auto raw = reinterpret_cast<::IInspectable**>(pointer);
 #pragma warning(pop)
 
-        // RoActivateInstance() will try to create an instance of the object,
-        // who's fully qualified name is the string in Name().
+        TerminalConnection::ITerminalConnection connection{ nullptr };
+
+        // A couple short-circuits, for connections that _we_ implement.
+        // Sometimes, RoActivateInstance is weird and fails with errors like the
+        // following:
         //
-        // The class has to be activatable. For the Terminal, this is easy
-        // enough - we're not hosting anything that's not already in our
-        // manifest, or living as a .dll & .winmd SxS.
+        /*
+        onecore\com\combase\inc\RegistryKey.hpp(527)\combase.dll!01234:
+        (caller: 01234) LogHr(2) tid(83a8) 800700A1 The specified
+        path is invalid.
+            Msg:[StaticNtOpen failed with
+            path:\REGISTRY\A\{A41685A4-AD85-4C4C-BA5D-A849ADBF3C40}\ActivatableClassId
+            \REGISTRY\MACHINE\Software\Classes\ActivatableClasses]
+        ...\src\cascadia\TerminalConnection\ConnectionInformation.cpp(47)\TerminalConnection.dll!01234:
+        (caller: 01234) LogHr(1) tid(83a8) 800700A1 The specified
+        path is invalid.
+            [...TerminalConnection::implementation::ConnectionInformation::CreateConnection(RoActivateInstance(name,
+            raw))]
+        */
         //
-        // When we get to extensions (GH#4000), we may want to revisit.
-        if (LOG_IF_FAILED(RoActivateInstance(name, raw)))
+        // So to avoid those, we'll manually instantiate these
```

Comment:
This addition seems kind of unrelated. Is there an issue for this?

Reply:
I should just get rid of that comment honestly. I think that was due to some wacky out-of-proc stuff I was doing for PMv2, and since we're _not doing that_ anymore, I don't think we'll run into that anymore.